src/cljs/nr/gamelobby.cljs

```diff
@@ -385,9 +415,17 @@
          nil)))
    [:div.button-bar
     [:div.rooms
-     [room-tab user s games "tournament" (tr [:lobby.tournament "Tournament"])]
+     [:div#filter.dropdown
+      [:a.dropdown-toggle {:href "" :data-toggle "dropdown"}
+       "Filter"
+       [:b.caret]]
+       [:div.dropdown-menu.blue-shade
+        (doall (for [[k] slug->format]
+                 ^{:key k}
+                 [format-toggle k (contains? visible-formats k)]))]]
+     [room-tab user s games "casual" (tr [:lobby.casual "Casual"])]
```

Comment:
Why change the order of the tabs?

Reply:
because of the change from float right to flexbox

Reply:
Cool, thank you.